src/main/java/com/corndel/cashregister/exercises/Exercise1a.java

```diff
@@ -17,7 +17,7 @@ public class Exercise1a {
    * both work, and which you use is up to your preference.
    */
   public static String getSummary(Recipe recipe) {
-    // TODO
-    return null;
+    return recipe.getName() + " has " + recipe.getIngredients().size() + " ingredients and serves " + recipe.getServings() + " people";
```

Comment:
Good use of string concatenation. 

Below is what it could look like with `String.format()`

```java
public static String getSummary(Recipe recipe) {
    return String.format("%s has %d ingredients and serves %d people", 
                         recipe.getName(), 
                         recipe.getIngredients().size(), 
                         recipe.getServings());
}

```